modules/configuration/src/main/java/org/apache/ignite/internal/configuration/util/ConfigurationSerializationUtil.java

```diff
@@ -200,7 +200,7 @@
                 buf.put(header);
 
                 for (int i = 0; i < strings.length; i++) {
-                    buf.putInt(strings[i].length());
+                    buf.putInt(strBytes[i].length);
```

Comment:
Can use regular loops instead of streams above?

Reply:
I believe that those streams will be inlined

Reply:
I am not aware of such an optimization.